.github/workflows/cypress.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 concurrency:
   group: cypress-${{ github.ref }}
-  cancel-in-progress: true
+  cancel-in-progress: false
```

Comment:
wanted?

Reply:
Yes I think it is annoying when the status get red because it was cancelled by the next commit. I think for this project it is ok to wait for the workflows to complete before running the other ones